packages/plugins/job-proposal-ui/src/lib/proposal/components/proposal/proposal.component.scss

```
@@ -1,23 +1,35 @@
 @forward '@shared/_pg-card';
 
+// Header-band statistics: three compact stat cards (big value over a muted
```

Stylelint (17.14.0)

[error] Unknown word Header-band (CssSyntaxError)

(CssSyntaxError)